src/plugins/cachefilter/cachefilter.c

```diff
@@ -42,7 +42,7 @@ int elektraCachefilterGet (Plugin * handle, KeySet * returned, Key * parentKey)
 	void * cache = elektraPluginGetData (handle);
 	if (cache == NULL)
 	{
-		toBeCached = ksNew (ksGetSize (returned), KS_END);
+		toBeCached = ksNew (0, KS_END);
```

Comment:
> No I didn't but this change is also not crucial.

Thus I wondered why you changed it. Comments help in such situations.

But you are right, lets move forward for now. Before you start changing `ksNew` (to be suitable for mmap), it would be great to have benchmarks for such reallocations.